src/System.ComponentModel.TypeConverter/src/System/ComponentModel/MemberDescriptor.cs

```diff
@@ -327,9 +345,8 @@ private void FilterAttributesIfNeeded()
                     {
                         FillAttributes(list);
                     }
-                    catch (Exception e)
+                    catch (Exception)
                     {
-                        Debug.Fail($"{_name}>>{e}");
```

Comment:
Why are you removing this?

Reply:
Can be triggered and the tests I wrote did

Reply:
How would it be triggered? What is the exception thrown?

Reply:
This is in the test `Attributes_GetWithThrowingFillAttributes_ReturnsExpected`

```cs
[Fact]
public void Attributes_GetWithThrowingFillAttributes_ReturnsExpected()
{
    var descriptor = new ThrowingFillAttributesMemberDescriptor("Name");
    Assert.Equal(new Attribute[] { descriptor.Attribute }, descriptor.Attributes.Cast<Attribute>());
}

[Fact]
public void AttributeArray_GetWithThrowingFillAttributes_ReturnsExpected()
{
    var descriptor = new ThrowingFillAttributesMemberDescriptor("Name");
    Assert.Equal(new Attribute[] { descriptor.Attribute }, descriptor.AttributeArray);
}

private class ThrowingFillAttributesMemberDescriptor : MemberDescriptor
{
    public ThrowingFillAttributesMemberDescriptor(string name) : base(name)
    {
    }

    public Attribute Attribute { get; } = new MockAttribute1();

    public new Attribute[] AttributeArray
    {
        get => base.AttributeArray;
        set => base.AttributeArray = value;
    }

    protected override void FillAttributes(IList attributeList)
    {
        attributeList.Add(Attribute);
        throw new NotImplementedException();
    }
}
```

Reply:
Old behaviour: release no exception thrown - it is caught. debug assert
New behaviour: release/debug no exception thrown - it is caught